main.py

```diff
@@ -977,8 +977,7 @@ def get_redirect_route(
         r'%s' % feconf.TASK_URL_FEEDBACK_STATUS_EMAILS,
         tasks.FeedbackThreadStatusChangeEmailHandler),
     get_redirect_route(
-        r'%s' % feconf.TASK_URL_DEFERRED,
-        tasks.DeferredTasksHandler),
+        r'%s' % feconf.TASK_URL_DEFERRED, tasks.DeferredTasksHandler),
```

Comment:
Any specific reason for this change? (I'm code owner to only this file)

Reply:
Just that it fits on one line. Not needed at all I just forgot that I changed this.